doc/source/tutorials.rst

```diff
@@ -0,0 +1,278 @@
+.. _tutorials:
+
+Using Braket Client
+-------------------
+
+
+.. _register_account:
+
+Registering for an AWS Account
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The usage of the Amazon Braket LocalSimulator does not require an account. However, to use Amazon Braket devices such as the statevector simulator and hardware, the user needs to register for an account here: https://signin.aws.amazon.com/signup?request_type=register.
+
+The user also needs to ensure the prerequisites have been configured. More information can be found here: https://github.com/amazon-braket/amazon-braket-sdk-python#prerequisites.
+
+
+.. _execute_a_circuit:
+
+Submit a circuit
+^^^^^^^^^^^^^^^^
+
+In this example, we will demonstrate how to submit a Qibo circuit onto an Amazon Braket device. We will use IQM Garnet as an example. Note that by default `verbatim_circuit=False`, therefore the transpilation of the input circuit and assignment of the best qubits to use will be left to the device.
+
+To use BraketClientBackend, we import `BraketClientBackend` from `qibo_cloud_backends.braket_client`:
+
+.. code-block:: python
+
+   from qibo_cloud_backend.braket_client import BraketClientBackend
+
+Then, construct a Qibo circuit.
+
+.. code-block:: python
+
+   from qibo import gates, Circuit as QiboCircuit
+
+   c = QiboCircuit(2)
+   c.add(gates.H(0))
+   c.add(gates.CNOT(0, 1))
+   c.add(gates.M(0))
+   c.add(gates.M(1))
+
+   print(c.draw())
+
+We should get this circuit:
+
+.. code-block:: python
+
+   q0: -H-o-M-
+   q1: ---x-M-
+
+Now, we initialize a Amazon Braket device (e.g. IQM Garnet) and execute circuit `c` on the backend `AWS`.
+
+.. code-block:: python
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   AWS = BraketClientBackend(device = device, verbatim_circuit=False)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+Alternatively, one can also use the LocalSimulator to execute circuit `c`. This can be done by leaving `device` empty as it defaults to the LocalSimulator.
+
+.. code-block:: python
+
+   AWS = BraketClientBackend(verbatim_circuit=False)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+To monitor the status of a circuit that is executed, especially on an Amazon Braket device, one can set `monitoring=True`. By default, `monitoring=False`.
+
+.. code-block:: python
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   AWS = BraketClientBackend(device = device, verbatim_circuit=False, monitoring=True)
```

Comment:
```suggestion
   AWS = BraketClientBackend(device = device, verbatim_circuit=False, verbosity=True)
```

Reply:
I forgot to change the documentation. Will do that!